.gitignore

```diff
@@ -288,3 +288,6 @@ cover/*
 .venv*/
 *.iml
 !/scripts/*
+
+example_config.yaml
+bilara-data/
```

Comment:
I added these two since when developing locally, you need these but don't want them placed under version control.